### PR TITLE
Downloadpage

### PR DESCRIPTION
**Pull request by StephanieGalindo10**
Finished Download page

### CODE REVIEW

File: download.html [hunk 1]
```diff
@@ -34,30 +34,80 @@
...
 
                     </div>
                     <div class="paragraph">
-                    <h1> A text editor can be used for many tasks including editing and writing code. There are many text editors out there but we recommend Atom or Visual Studio Code as they are both free and commonly used by a lot of developers.</h1>
+                    <p> A text editor can be used for many tasks including editing and writing code. There are many text editors out there but we recommend Atom or Visual Studio Code as they are both free and commonly used by a lot of developers.</p>
                 </div>
                 <div class="w3-container">
-                <div class="rectangle"></div>
-              <!--<div class="rectangle2"></div>--->
+                <div class="rectangle">
```
Comment (Kaleidics) [inline download.html]: rectangle is not descriptive enough as a class, please reference the BEM in the readme

File: download.html [hunk 2]
```diff
@@ -34,30 +34,80 @@
...
+                <div class="rectangle"> 
+                    <img id="green" src="assets/atom.png" alt="">
+               
+                    <h3>Atom</h3> 
+                    <br> 
+                    <a href="" class="learn">Download</a>
+                </div>
+                
+              <div class="rectangle">
+                  <img id="blue" src="assets/vscode.png" alt="">
```
Comment (Kaleidics) [inline download.html]: What is `id` blue used for? If it is used to style, please use a modifier class. `id` should be unique, blue is not a unique identifier if used to style for the color blue.

File: download.html [hunk 3]
```diff
@@ -34,30 +34,80 @@
...
                 </header>
                     </div>
-
+                    <div class="wrapper">
 
                     <div class="boxed">
                         The download links below will take you to the product’s download page. Downloads are never hosted on our servers.
                       </div>
                 
                     <div class="Text-Editors">
```
Comment (Kaleidics) [inline download.html]: Please use a more consistent naming case

File: download.html [hunk 4]
```diff
@@ -34,30 +34,80 @@
...
+               
+                    <h3>Atom</h3> 
+                    <br> 
+                    <a href="" class="learn">Download</a>
+                </div>
+                
+              <div class="rectangle">
+                  <img id="blue" src="assets/vscode.png" alt="">
+               
+                <h3>Visual Studio Code</h3>
```
Comment (Kaleidics) [inline download.html]: Please use ordered headings to maintain correct accessibility and SEO. You can style the heading with class modifier if the default styles doesn't fit the design.

File: download.html [hunk 5]
```diff
@@ -34,30 +34,80 @@
...
+                         <img id="white" src="assets/slack.png" alt="">
+                         <h2>Slack</h2> 
+                         <br>
+                         <a href="" class="learn">Download</a>
+                     </div>
+                     
+            </div>
+<footer>
+    <p>Interested in joining our group?</p> 
+    <br>
```
Comment (Kaleidics) [inline download.html]: Using breaks for spacing is ok. But I would recommend using css to style breaks unless the breaks are specifically break text within an existing element.

File: download.html (hunk 2 above)
Comment (StephanieGalindo10) [reply download.html]: The blue is for the logo color of vscode. The images given by jeff only has the logo so I had to do it all in css

File: download.html (hunk 1 above)
Comment (StephanieGalindo10) [reply download.html]: Okay will change it

File: download.html (hunk 3 above)
Comment (StephanieGalindo10) [reply download.html]: Will change it

File: download.html (hunk 5 above)
Comment (StephanieGalindo10) [reply download.html]: I user br for spacing

File: download.html (hunk 4 above)
Comment (StephanieGalindo10) [reply download.html]: Will change

**Comment by StephanieGalindo10** [comment]
Will fix them and show on Friday

File: download.html [hunk 6]
```diff
@@ -54,31 +54,31 @@ <h1> Text Editors</h1>
...
+            
+                <a href="https://code.visualstudio.com/" target="_blank" class="learn">Download <img class="pointer"src="assets/arrow.png"></a></div>
 
                 </div>
                 <div class="learnmore">
-                    <p>Want to know the differences between Atom and Visual Studio Code? Click on learn more for more information.</p> <br><br>
+                    <p>Want to know the differences between Atom and Visual Studio Code? Click on learn more for more information.</p> 
 
-                    <a href="https://simpledev.io/" class="learn">Learn More <img class="pointer"src="assets/arrow.png"></a>
+                    <a href="https://simpledev.io/" target="_blank" class="learn">Learn More <img class="pointer"src="assets/arrow.png"></a>
```
Comment (Kaleidics) [inline download.html]: One more small change for `a` add the `rel="noopener noreferrer"` attribute for `a` that open in a new tab

**Comment by StephanieGalindo10** [comment]
Please review updated infor let me know what needs to be change having  fun in css

**Comment by StephanieGalindo10** [comment]
add the new code for href links

**Pull request merged**